12.nadeo-importer/03.how-to-make-a-specularmap-diffusemap-for-objects/docs.md

```diff
@@ -29,11 +49,13 @@ For selfmade textures on objects we need _TDNS_ textures:
 
 ![](http://trackmania-carpark.com/imagespark/up4/52cea5bd6a098.jpg)
 
-* DXT1 RGB 4bpp no alpha
+* DXT1 RGB 4bpp no alpha (care exceptions)
```

Comment:
"Care exceptions"? What does that mean?

Reply:
Some special files have different settings, there for i wrote the info (care exceptions) this settings doesnt work for those, you should always compare/take a look onto original files format.

All that is the reason im upset about nadeo/ubisoft missing informations everywhere, make it hard for modders to mod smth. without knowledge or a lot of time to search for needed infos. Thats in my opinion also one of teh reasons TM died slowly/the player base is smaller.

Reply:
Okay, well either way take a look at my other comment below. You can make this "care exceptions" text a lot more concise.